app/common_functions/read_all.py

```diff
@@ -0,0 +1,11 @@
+
+
+def read_all(model):
```

Comment:
I see that you didn't use this helper in your code. Per the discussion Makhabat and I had in our 1:1, I think this could be used to DRY up your code. Nice work.